web/components/commons/view_mixins.py

```diff
@@ -33,7 +33,7 @@ def run_pipeline(self, with_permalinks=False):
         except Exception as e:
             pipeline = None
             data['report']['error_title'] = e.__class__.__name__
-            data['report']['error_message'] = e.msg
+            data['report']['error_message'] = str(e)
```

Comment:
This breaks Py2/3 compatibility. Should use `compat.str` although I note there are places that may not be possible, like in url definitions ([ref.](https://github.com/frictionlessdata/goodtables.io/blob/master/web/components/api/__init__.py#L15)) (is this one?)